rsocket/RSocketServer.cpp

```diff
@@ -143,7 +142,10 @@ void RSocketServer::onRSocketSetup(
       std::move(connectionParams.connectionEvents),
       nullptr, /* resumeManager */
       nullptr /* coldResumeHandler */);
-  connectionManager_->manageConnection(rs, *eventBase);
+
+  connectionSet_->insert(rs, eventBase);
+  rs->registerSet(connectionSet_);
```

Comment:
I could fold all these calls to `registerSet()` into `ConnectionSet::insert()`, but that'd mean `ConnectionSet` would have to inherit from `enable_shared_from_this`, something which I don't really feel like doing....